roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -36,13 +44,6 @@
         - ansible_facts.services[item] is defined
         - ansible_facts.services[item]["status"] != "not-found"
 
-    - name: Ensure firewalld service is disabled
```

Comment:
I'd prefer to preserve this disable task instead of moving it above to avoid touching firewalld if we are not going to enable nftables.